poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFParagraph.java

```diff
@@ -488,11 +488,11 @@ public String getFootnoteText() {
      * specified in the style hierarchy, then no alignment is applied to the
      * paragraph.
      *
-     * @return the paragraph alignment of this paragraph.
+     * @return the paragraph alignment of this paragraph. Can be null if not set.
```

Comment:
This is a breaking change. I know you don't like the way things work but unfortunately, in a 20 year old project with lots of users, we can't just change API behaviours. We have no major releases planned, just patch releases - so if you want to break the API, this change will need to wait.

Can you instead consider adding a new method `public boolean isAlignmentSet()`? New methods won't cause backward compatibility issues.

Reply:
Thanks a lot for the insight! You are totally right, backwards API compatibility must be considered, and a method that starts returning null can cause a lot of headaches... I will implement this behavior as a new method instead.

What do you think about allowing `setAlignment(null)`? It sounds less dangerous from the compatibility POV, but maybe it looks a bit weird? Better with a new method `unsetAlignment()`?